std/stdio.d

```diff
@@ -498,17 +498,22 @@ struct File
     private Impl* _p;
     private string _name;
 
-    package this(FILE* handle, string name, uint refs = 1, bool isPopened = false) @trusted
+    package this(FILE* handle, string name, uint refs = 1, bool isPopened = false) @trusted @nogc nothrow
     {
         import core.stdc.stdlib : malloc;
         import std.exception : enforce;
```

Comment:
This import is now unused and can now be removed.

Reply:
good catch!